<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTypeStatsWindow</class>
 <widget class="QDialog" name="FileTypeStatsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Type Statistics</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headingLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="toolTip">
        <string>Update the statistics for the same url</string>
       </property>
       <property name="text">
        <string>&amp;Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="locateButton">
         <property name="toolTip">
          <string>Show a list of directories containing files with this type and suffix</string>
         </property>
         <property name="text">
          <string>&amp;Locate by Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sizeStatsButton">
         <property name="toolTip">
          <string>Show a size statistics window for the currently-selected type and suffix</string>
         </property>
         <property name="text">
          <string>&amp;Size Statistics</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="syncCheckBox">
       <property name="toolTip">
        <string>&lt;p style='white-space:pre'&gt;If checked: when a different directory is selected in the main&lt;br/&gt;window, the content of this window is automatically updated&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Sync with &amp;main window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionLocate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/magnifier-left.png</normaloff>:/icons/magnifier-left.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Locate</string>
   </property>
   <property name="shortcut">
    <string>Enter</string>
   </property>
  </action>
  <action name="actionSizeStats">
   <property name="text">
    <string>Si&amp;ze Statistics</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileTypeStatsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSizeStats</sender>
   <signal>triggered()</signal>
   <receiver>sizeStatsButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>refreshButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
